src/server_manager/web_app/gcp_account.ts

```diff
@@ -169,12 +169,12 @@ export class GcpAccount implements gcp.Account {
   async listBillingAccounts(): Promise<BillingAccount[]> {
```

Comment:
Could you rename the method to `listOpenBillingAccounts()` for clarity?

Reply:
Done